flowr/src/main/java/com/fueled/flowr/Flowr.java

```diff
@@ -740,9 +784,11 @@ public Builder noTransactionAnimation() {
 
         /**
          * Displays the fragment using this builder configurations.
+         *
+         * @return id Identifier of the stated to be popped.
```

Comment:
Kindly modify the documentation to Identifier returned by `transaction.commit()`, to be used along with `#closeUptoWithResults()`

Reply:
Done! Thanks 👍